assets/src/components/mapPage.tsx

```diff
@@ -324,6 +324,10 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
             />
           )}
         </div>
+        <button
+          onClick={toggleSearchDrawer}
+          className="c-map-page__panel-backdrop-button"
+        ></button>
```

Comment:
I'm not a fan of this, I would've preferred to add a pseudo element to the `<DrawerTab/>` that stretched it's clickable area when on certain screen sizes and states, but this was easiest since I couldn't get that to work.

Reply:
**suggestion:** I think this could be a self-closing tag.

Reply:
> **suggestion:** I think this could be a self-closing tag.

ebe5948b2a594acfefb5a8fee3fd70637df523a1